cmake/draco_install.cmake

```diff
@@ -109,4 +89,36 @@ macro(draco_setup_install_target)
     install(TARGETS draco_maya_wrapper DESTINATION "${libs_path}")
   endif()
 
+  # pkg-config: draco.pc
+  configure_file("${draco_root}/cmake/draco.pc.template"
+                 "${draco_build}/draco.pc" @ONLY NEWLINE_STYLE UNIX)
+  install(FILES "${draco_build}/draco.pc" DESTINATION "${libs_path}/pkgconfig")
+
+  # CMake config: draco-config.cmake
+  configure_package_config_file(
+    "${draco_root}/cmake/draco-config.cmake.template"
+    "${draco_build}/draco-config.cmake"
+    INSTALL_DESTINATION "${data_path}/cmake/draco")
+
+  write_basic_package_version_file(
+    "${draco_build}/draco-config-version.cmake"
+    VERSION ${DRACO_VERSION}
+    COMPATIBILITY AnyNewerVersion)
+
+  export(
+    EXPORT dracoExport
+    NAMESPACE draco::
+    FILE "${draco_build}/draco-targets.cmake")
+
+  install(
+    EXPORT dracoExport
+    NAMESPACE draco::
+    FILE draco-targets.cmake
```

Comment:
Now that the `draco-config.cmake.template` is essentially empty, this could also be named `draco-config.cmake` directly, thereby removing the whole `configure_package_config_file` step above and its associated parts.

Reply:
For current master branch yes.
In the meantime, it scales better to keep targets definition outside of `draco-config.cmake` (external dependencies may be added -requiring find_dependency()-, components or whatever logic).

Reply:
I know, just providing it as an option for the Draco maintainers. I am personally in favour of removing dead code. After all, by the time Draco needs custom draco-config logic the CMake the mechanism for achieving that could have changed already.